source/_data/data.json

```diff
@@ -2,7 +2,7 @@
   "pl": true,
   "BUILD_TARGET": "pl",
   "paths": {
-    "pl_theme_root": "../../../.."
+    "assets": "../../../assets"
```

Comment:
So if you wanted to move the build target of the assets, you'd edit this here?

Reasonable to consider switching this to a yaml file?

Reply:
Yes, this should get updated if/whenever the build target of the assets changes. But this is a global pattern lab helper, the actual build target is set inside webpack.shared.config.js (the key titled "output"). I imagine drupal-theme.js or the .theme file will set `path.assets` to `\Drupal::theme()->getActiveTheme()->getPath();`


To the second point, I would like to see all of source/_data converted to .yml eventually, although I don't know if that's an option in the case of `scssVariables.json` and how that's being generated. Right now our practices aren't consistent in either direction.

Reply:
There is a ticket to convert all human-created _data files to yaml. Robot-created _data should stay json. More than a few _data files should be deleted or consolidated.